### ISSUE TITLE
Missing package

### ISSUE THREAD

**Issue by tbowenbw** [Bandwidth/shared-components#215]
Currently, if I try to add shared-components to a new create-react-app, I get the following:
```
Module not found: Can't resolve 'lodash.throttle' in '/Users/tbowen/dev/interrobang-ui/node_modules/@bandwidth/shared-components/dist/behaviors/ScrollShadow'```

**Comment by a-type** [comment]
I feel like Lerna has messed up our dependencies or something...

**Comment by zelitzsi** [comment]
Fixed in #224.
